Extensions/CGActivePlayer.js

```diff
@@ -0,0 +1,18 @@
+import objMgr, {me} from "../Core/ObjectManager";
+
+
+Object.defineProperties(wow.CGActivePlayer.prototype, {
+
+  currentParty: {
+    /**
+     * Get the current active party if it exists, or return `undefined` if not in a party.
+     * @returns {wow.Party | undefined} - Returns the current party instance or undefined.
+     */
+    get: function () {
+      return (wow.Party?.currentParty) ? wow.Party.currentParty : undefined;
```

Comment:
`wow.Party` is a builtin class and will always exist, `wow.Party.currentParty` already returns undefined if player is not in a party.

Hence `wow.Party.currentParty` is enough

Reply:
Done.